app/src/main/java/com/greenfox/tribesoflagopusandroid/AppModule.java

```diff
@@ -24,7 +22,8 @@
 public class AppModule {
 
     private Context context;
-    private Boolean switchLoginOrMockService = true;
+    private static Boolean isLoginServiceActive = true;
```

Comment:
@georgezsiga sorry, it should be `final` too, my mistake. and also naming:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names